src/api/graphql/datasources/MessageRepo.js

```diff
@@ -11,21 +11,19 @@ class MessageRepo extends BaseRepo {
 
   async getAll({ page, limit }) {
     try {
-      const responses = await this.model
+      return this.model
         .find()
         .skip(page)
         .limit(limit);
-      return responses;
     } catch (error) {
       throw error;
     }
   }
   async getGroupChat({ page, limit }) {
     try {
-      const responses = await GroupMessage.find()
+      return GroupMessage.find()
```

Comment:
We shouldn't use others model inside a repository. Each repository mapping with a model. If you want to use a query from others model. Please inject others repository in constructor. Read more https://medium.com/better-programming/what-is-dependency-injection-b2671b1ea90a